docs/how-tos/hello-world-module.md

```diff
@@ -390,13 +494,197 @@ You don't need to edit any of the validate or configuration methods because you'
 
 {{< /expand >}}
 
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+### Implement the camera API
+
+First, implement the camera API methods by editing the camera class definition:
+
+1. Add the following to the list of imports at the top of <file>hello-world/models/hello-camera.go</file>:
+
+   ```go {class="line-numbers linkable-line-numbers" data-start="6"}
+   "os"
+   "reflect"
+   "io/ioutil"
+   ```
+
+1. Add `imagePath = ""` to the global variables so you have the following:
+
+   ```go {class="line-numbers linkable-line-numbers" data-line="22" data-start="19" data-line-offset="19"}
+   var (
+       HelloCamera      = resource.NewModel("jessamy", "hello-world", "hello-camera")
+       errUnimplemented = errors.New("unimplemented")
+       imagePath        = ""
+   )
+   ```
+
+1. In the test script you hard-coded the path to the image.
+   For the module, let's make the path a configurable attribute so you or other users of the module can set the path from which to get the image.
+
+   Edit the `type Config struct` definition, replacing the comments with the following:
+
+   ```go {class="line-numbers" data-start="33"}
+   type Config struct {
+       resource.AlwaysRebuild
+       ImagePath string `json:"image_path"`
+   }
+   ```
+
+   This adds the `image_path` attribute and causes the resource to rebuild each time the configuration is changed.
+
+1. Delete the `Reconfigure()` function entirely, since the camera will rebuild instead of reconfiguring.
+
+1. We are not providing a default image but rely on the end user to supply a valid path to an image when configuring the resource.
+   This means `image_path` is a required attribute.
+   Add the following code to the `Validate` function to throw an error if `image_path` isn't configured or isn't a string:
```

Comment:
```suggestion
   Replace the `Validate` function with the following function to throw an error if `image_path` isn't configured or isn't a string:
```

Reply:
```suggestion
   Replace the `Validate` function with the following code to throw an error if `image_path` isn't configured or isn't a string:
```
Okay yeah. Changing to "code" so as not to imply it's a totally different function.